pkg/scheduler/cache/node_info_test.go

```diff
@@ -240,6 +241,43 @@ func TestSetMaxResource(t *testing.T) {
 	}
 }
 
+type testingMode interface {
```

Comment:
copy from `pkg/scheduler/internal/cache`.

Reply:
I've never noticed this interface before. Honestly, I would rather get rid of it and have `makeBasePod` take a `*testing.T` arg. Since it's copy-paste, I don't mind leaving it as-is for the current PR.